docs/developer-docs/defi/tokens/token-standards.mdx

```diff
@@ -0,0 +1,197 @@
+---
+keywords: [intermediate, defi, overview, tokens, icrc-1, icrc-2, icrc-3]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Token standards
+
+<MarkdownChipRow labels={["Intermediate", "Concept"]} />
+
+## Overview
+
+Tokens deployed on ICP must adhere to a standard. The network's native token, ICP, adheres to its own standard as it is unique on the network in the sense that it is used for network utility and governance. Tokens created for other purposes on the network typically adhere to one of the ICRC standards.
+
+[ICRC](https://github.com/dfinity/ICRC) stands for `Internet Computer Request for Comments'. ICRC standards are proposed by the community working group and are not specific to tokens; they can be created for anything on ICP, such as services, canister calls, or authentication methods. However, this documentation will detail the ICRC standards that are specific to tokens.
```

Comment:
This paragraph repeats a similar paragraph on another page. Is this repeating intentional?

Reply:
Yes, this is intentional